plans/packit-integration.fmf

```diff
@@ -0,0 +1,20 @@
+discover:
+    how: fmf
+    url: https://github.com/packit/packit
+    filter: tier:0 | tier:1
+
+adjust:
+  - when: "how == integration"
+    because: "provide latest python-specfile rpm when running locally"
+    prepare+:
+      - name: python3-specfile rpm
+        how: install
+        directory: noarch/
```

Comment:
So, when running locally, one is supposed to build a rpm and put it in `noarch` directory?

Reply:
Ok, I see, `packit build locally` does that. I think we should document that somewhere (and probably also how to run other test plans).

Reply:
Yes, I was thinking what's the best place for this. In tmt, we used something like `<repo root>/tmp/RPMs`. But I found it quite convenient to just run `packit build locally` and then run the test without any need for moving things around. Or do you see a better place where to put RPMs to be installed when running tests locally?

Reply:
No, I think it's fine, we just need to document it.

Reply:
Done, I added a `CONTRIBUTING.md` describing this.